### ISSUE TITLE
Changing filamment after runout during print - adjust speed of extruder

### ISSUE THREAD

**Issue by ivantichy** [prusa3d/Prusa-Firmware#2310]
When filament runs-out during print user follows procedure to continue with a new filament. Happy scenario is OK but when you select that new filament was not loaded correctly extruder tries to load more filament. The problem is that this step contains two parts - firstly extruder goes pretty fast (assuming it is loading filament into empty hotend) and then it slows. This is OK for loading into empty hotend but when tried in situation where filament is already loaded this cause huge extruder skipping.

MK3 prusa3d_fw_MK3_3_8_0_2684.hex
PETG Mladec

**Comment by vintagepc** [comment]
While not experiencing this directly, I've noticed some filament change menus have 3 options, the 3rd being "it's loaded but the colour isn't right yet" which has the effect of purging more (but not at a high speed). I have also encountered change prompts where this 3rd option is not present and the answer is only yes/no. 

If your issue is on this "2-choice" change menu, I wonder if unifying them to all have the third choice would help fix this issue.
If you have the choice to pick the "colour not correct" option, does it also cause this skipping for you?
